cast/src/main.rs

```diff
@@ -74,9 +73,12 @@ async fn main() -> Result<()> {
                 &mut account,
             )
             .await?;
-            // todo: #2107
-            eprintln!("Class hash: {}", declared_contract.class_hash);
-            eprintln!("Transaction hash: {}", declared_contract.transaction_hash);
+
+            println!("Class hash: {:#x}", declared_contract.class_hash);
```

Comment:
Have you thought about making it possible to allow for other formats to be returned? it could be a flag.

Reply:
**https://github.com/software-mansion/protostar/pull/2135/commits/869286f056d40d2daf71aa17008694609e55503b**